index.html

```diff
@@ -4,10 +4,37 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">   
     <link rel="stylesheet" href="style.css">
     <title>Portfolio</title>
 </head>
 <body>
-    <div class="header"></div>
+    <div class="header">
+        <img src="/image/images.png" width="100px">
+        <a href="#" class="fa fa-bars"></a>
+    </div>
+
+    <div class="menu">
+        <ul class="menu">
+            <li><a href="#home" class="menu-btn">Home</a></li>
+            <li><a href="#about" class="menu-btn">About</a></li>
+            <li><a href="#services" class="menu-btn">Services</a></li>
+            <li><a href="#skills" class="menu-btn">Skills</a></li>
+            <li><a href="#contact" class="menu-btn">Contact</a></li>
+        </ul>
+    </div>
```

Comment:
- [ ] Nice work adding the menu :clap: However I will suggest to remove it. It not require in this milestone and don't worry :cat: You will learn how to implement it in the appropriate way in the upcoming milestone